connection-api/src/assembly/docs.xml

```diff
@@ -1,20 +1,18 @@
 <!--
-  ~
-  ~  The contents of this file are subject to the Terracotta Public License Version
-  ~  2.0 (the "License"); You may not use this file except in compliance with the
-  ~  License. You may obtain a copy of the License at
-  ~
-  ~  http://terracotta.org/legal/terracotta-public-license.
-  ~
-  ~  Software distributed under the License is distributed on an "AS IS" basis,
-  ~  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-  ~  the specific language governing rights and limitations under the License.
-  ~
-  ~  The Covered Software is Connection API.
-  ~
-  ~  The Initial Developer of the Covered Software is
-  ~  Terracotta, Inc., a Software AG company
-  ~
+Copyright Terracotta, Inc.
```

Comment:
The `license-maven-plugin` is somewhat strict about the whitespace needed in and around a copyright/license block.  Its rules for `xml` files are particularly restrictive.  Use `license:format` to make adjustments to this file so it will pass muster.  (You might ask "why doesn't it complain about the `pom.xml` files?"  Because the inherited configuration explicitly ignores them.)